### PR TITLE
adds medal printer flatpack to IAA closet and allows IAA to use it

### PR DESCRIPTION
**Pull request by toomykins**
adds a medal printer flatpack to IAA's legal closet
IAA will be able to use medal printers, lowering the req access.

:cl:
 * rscadd: Added medal printer flatpack to IAA closet, which they can now use.

### CODE REVIEW

**Comment by BurntDevil** [comment]
legal closets, are they also the ones in court rooms? might need to check to make sure they dont end up in there

**Comment by toomykins** [comment]
> legal closets, are they also the ones in court rooms? might need to check to make sure they dont end up in there

damn, you're right. this makes it a little more finicky...

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -75,6 +75,9 @@
 		/obj/item/clothing/shoes/brown,
 		/obj/item/clothing/shoes/black,
 	)
+	if(istype(loc.loc,/area/lawoffice))
```
Comment (DamianX) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:78]: This is just awful. The proper way to do this is to create and map a different subtype.

**Comment by toomykins** [comment]
not touching maps with this tgm shite

**Pull request closed without merging**